Sources/Core/AWSSDKIdentity/Sources/InternalAWSSTS/IdentityProvidingSTSClient.swift

```diff
@@ -0,0 +1,42 @@
+//
```

Comment:
Generated STS client file. This is the concrete implementation of `IdenttiyProvidingSTSClient` protocol. It constructs an instance of the internal STS client generated in `InternalAWSSTS` target, uses it to fetch creds, and returns `AWSCredentialIdentity`.